internal/cache/fastcache/fastcache.go

```diff
@@ -0,0 +1,46 @@
+package fastcache
+
+import (
+	"bytes"
+	"encoding/gob"
+	"github.com/caos/zitadel/internal/errors"
+
+	"github.com/VictoriaMetrics/fastcache"
+)
+
+type Fastcache struct {
+	cache *fastcache.Cache
+}
+
+func NewFastcache(config *Config) (*Fastcache, error) {
+	return &Fastcache{
+		cache: fastcache.New(config.MaxCacheSizeInByte),
+	}, nil
+}
+
+func (fc *Fastcache) Set(key string, object interface{}) error {
+	var b bytes.Buffer
+	enc := gob.NewEncoder(&b)
```

Comment:
create encoder in `NewFastcache` instead of every `Set`

Reply:
Problem is, that I need the buffer to set